guardian/gateway
# Test adding different newsletters into the onboarding flow

## What does this change?

Replaces #1282, and currently branched off `mh/update-onboarding-flow`. This sets up an A/B test where 50% of users will see different newsletter options when going through the onboarding flow.



## How to test

Set a `GU_mvt_id cookie` to `1` to see the changes being tested. Set it to `2` to see the control group behaviour, which shows the same list as is currently used.

Unsure how best to test region-specific behaviour (different regions have different newsletters). Locally I have been hardcoding the relevant region to test.



## How can we measure success?



I suppose we look at which variant leads to the most signups?

## Have we considered potential risks?



## Images



### Control (US)

![image](https://user-images.githubusercontent.com/3338808/151358797-046574e2-9c16-4a7a-bfc4-2547eef300aa.png)

### Variant (US)

![image](https://user-images.githubusercontent.com/3338808/151362150-2a03515e-528d-4f3e-bbab-dc18a135da33.png)

## Accessibility
